src/main/java/uk/gov/cslearning/record/service/UserRecordService.java

```diff
@@ -111,4 +117,21 @@ public void setUserDepartmentAndProfession(String userId, Collection<CourseRecor
             courseRecordRepository.saveAll(courseRecords);
         }
     }
+
+    @Transactional
+    public void deleteUserRecords(String uid) {
+        statementsRepository.deleteAllByLearnerUid(uid);
+        courseRecordRepository.deleteAllByUid(uid);
+    }
+
+    @Transactional
+    public Collection<Statements> listStatements() {
+        return statementsRepository.findAll();
+    }
+
+    @Transactional
+    public void deleteOldStatements() {
+        DateTime dateTime = DateTime.now().minusYears(3);
```

Comment:
DOD - These needs to be months and come from configuration.
Where is the scheduler bit to trigger the changes